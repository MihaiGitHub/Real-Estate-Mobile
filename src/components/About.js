import React, { Component } from 'react';
import { ScrollView, Text, View, Button } from 'react-native';

class About extends Component {

    render() {

        return (
            <View>
                <Text>Title: Home Square Real Estate</Text>
                <Text>Version: 1.0</Text>
                <Text>Website: https://naszpolskidom.azurewebsites.net/</Text>
                <Text>
                    Home Square helps you buy, sell or rent apartments and houses. Our
                    real estate agents make the entire process easy from beginning to end.
                </Text>
            </View>
        );
    }
}

export default About;