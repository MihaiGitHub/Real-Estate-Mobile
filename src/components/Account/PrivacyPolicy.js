import React from "react";
import { Text, View } from "react-native";

export function PrivacyPolicy({ navigation }) {
  return (
    <View>
      <Text>privacy policy</Text>
    </View>
  );
}
