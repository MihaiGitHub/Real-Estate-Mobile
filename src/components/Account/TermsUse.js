import React from "react";
import { Text, View } from "react-native";

export function TermsUse({ navigation }) {
  return (
    <View>
      <Text>terms use</Text>
    </View>
  );
}
