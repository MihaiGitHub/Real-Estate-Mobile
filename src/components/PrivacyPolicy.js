import React, { Component } from 'react';
import { ScrollView, Text, View, Button } from 'react-native';

class PrivacyPolicy extends Component {

    render() {

        return (
            <View>
                <Text>Privacy Policy</Text>
            </View>
        );
    }
}

export default PrivacyPolicy;