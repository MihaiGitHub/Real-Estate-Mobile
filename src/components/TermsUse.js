import React, { Component } from 'react';
import { ScrollView, Text, View, Button } from 'react-native';

class TermsUse extends Component {

    render() {

        return (
            <View>
                <Text>Terms of Use</Text>
            </View>
        );
    }
}

export default TermsUse;