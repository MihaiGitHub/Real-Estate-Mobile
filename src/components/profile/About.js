import React, { Component } from "react";
import { Container, Header, Content, Card, CardItem, Text, Body } from "native-base";

class About extends Component {

    render() {
        return (
            <Container>
                <Content padder>
                <Card>
                    <CardItem header bordered>
                        <Text style={styles.text2}>Title: Home Square Real Estate</Text>
                    </CardItem>
                    <CardItem bordered>
                    <Body>
                        <Text style={styles.text}>
                            Home Square helps you buy, sell or rent apartments and houses. 
                            Our real estate agents make the entire process easy from beginning to end.
                        </Text>
                        <Text style={styles.text2}>Website:</Text>
                        <Text style={styles.text2}>https://naszpolskidom.azurewebsites.net/</Text>
                    </Body>
                    </CardItem>
                    <CardItem footer bordered>
                    <Text style={styles.text2}>Version: 1.0</Text>
                    </CardItem>
                </Card>
                </Content>
            </Container>
        );
    }
}

const styles = {
    text: {
        fontFamily: 'OpenSans-Regular',
        marginBottom: 5 
    },
    text2: {
        fontFamily: 'OpenSans-Regular' 
    }
}

export default About;