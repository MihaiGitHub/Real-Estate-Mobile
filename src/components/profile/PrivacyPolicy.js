import React, { Component } from "react";
import { Container, Header, Content, Card, CardItem, Text, Body } from "native-base";

class PrivacyPolicy extends Component {

    render() {
        return (
            <Container>
                <Content padder>
                <Card>
                    <CardItem header bordered>
                        <Text style={styles.text}>Privacy Policy</Text>
                    </CardItem>
                    <CardItem bordered>
                    <Body>
                        <Text style={styles.text}>
                        We may use, store, and process personal information to (1) provide, understand, improve, and develop the HomeSquare Platform, 
                        (2) create and maintain a trusted and safer environment and (3) provide, personalize, measure, and improve our advertising and marketing.</Text>
                    </Body>
                    </CardItem>
                    <CardItem footer bordered>
                    <Text></Text>
                    </CardItem>
                </Card>
                </Content>
            </Container>
        );
    }
}

const styles = {
    text: {
        fontFamily: 'OpenSans-Regular'
    }
}

export default PrivacyPolicy;