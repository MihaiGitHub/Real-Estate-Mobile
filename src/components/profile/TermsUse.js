import React, { Component } from "react";
import { StyleProvider, Container, Header, Content, Card, CardItem, Text, Body } from "native-base";
//import getTheme from '../../native-base-theme/components/index';
//import material from './native-base-theme/variables/platform';

class TermsUse extends Component {

    render() {
        return (
            <Container>
                <Content padder>
                <Card>
                    <CardItem header bordered>
                        <Text style={styles.text}>Terms of Use</Text>
                    </CardItem>
                    <CardItem bordered>
                    <Body>
                        <Text style={styles.text}>
                        The HomeSquare Platform is an online marketplace that enables users to publish properties for sale or return
                        </Text>
                    </Body>
                    </CardItem>
                    <CardItem footer bordered>
                    <Text></Text>
                    </CardItem>
                </Card>
                </Content>
            </Container>
        );
    }
}

const styles = {
    text: {
        fontFamily: 'OpenSans-Regular'
    }
}

export default TermsUse;