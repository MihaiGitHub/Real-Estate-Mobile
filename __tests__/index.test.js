import React from "react";
import renderer from "react-test-renderer";
import About from "../src/components/profile/About";
import PrivacyPolicy from "../src/components/profile/PrivacyPolicy";
import TermsUse from "../src/components/profile/TermsUse";

it("About component renders correctly", () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("PrivacyPolicy component renders correctly", () => {
  const tree = renderer.create(<PrivacyPolicy />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("TermsUse component renders correctly", () => {
  const tree = renderer.create(<TermsUse />).toJSON();
  expect(tree).toMatchSnapshot();
});
