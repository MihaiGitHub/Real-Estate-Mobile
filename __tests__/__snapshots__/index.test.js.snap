// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      automaticallyAdjustContentInsets={false}
      contentContainerStyle={
        Array [
          Object {
            "padding": 10,
          },
          undefined,
        ]
      }
      contentInset={
        Object {
          "bottom": 0,
        }
      }
      enableAutomaticScroll={true}
      enableOnAndroid={false}
      enableResetScrollToCoords={true}
      extraHeight={75}
      extraScrollHeight={0}
      getScrollResponder={[Function]}
      handleOnScroll={[Function]}
      keyboardDismissMode="interactive"
      keyboardOpeningTime={250}
      keyboardShouldPersistTaps="handled"
      keyboardSpace={0}
      onScroll={[Function]}
      padder={true}
      resetKeyboardSpace={[Function]}
      resetScrollToCoords={
        Object {
          "x": 0,
          "y": 0,
        }
      }
      scrollEventThrottle={1}
      scrollForExtraHeightOnAndroid={[Function]}
      scrollIntoView={[Function]}
      scrollToEnd={[Function]}
      scrollToFocusedInput={[Function]}
      scrollToPosition={[Function]}
      showsVerticalScrollIndicator={true}
      style={
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        }
      }
      update={[Function]}
      viewIsInsideTabBar={false}
    >
      <View>
        <View
          style={
            Object {
              "backgroundColor": "#fff",
              "borderBottomWidth": 0.5,
              "borderColor": "#ccc",
              "borderLeftWidth": 0.5,
              "borderRadius": 2,
              "borderRightWidth": 0.5,
              "borderTopWidth": 0.5,
              "elevation": 3,
              "flexWrap": "nowrap",
              "marginBottom": 5,
              "marginLeft": 2,
              "marginRight": 2,
              "marginTop": 5,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.1,
              "shadowRadius": 1.5,
            }
          }
        >
          <View
            bordered={true}
            header={true}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderBottomWidth": 0.5,
                "borderColor": "#ccc",
                "borderRadius": 2,
                "flexDirection": "row",
                "paddingBottom": 15,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 15,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#007aff",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "600",
                }
              }
            >
              Title: Our House Real Estate
            </Text>
          </View>
          <View
            bordered={true}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderBottomWidth": 0.5,
                "borderColor": "#ccc",
                "borderRadius": 2,
                "flexDirection": "row",
                "paddingBottom": 10,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 10,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "flex-start",
                  "alignSelf": "stretch",
                  "flex": 1,
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#000",
                      "fontFamily": "System",
                      "fontSize": 16,
                    },
                    Object {
                      "marginBottom": 5,
                    },
                  ]
                }
              >
                Our House helps you buy, sell or rent apartments and houses. Our real estate agents make the entire process easy from beginning to end.
              </Text>
              <Text
                style={
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                  }
                }
              >
                Website:
              </Text>
              <Text
                style={
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                  }
                }
              >
                https://naszpolskidom.azurewebsites.net/
              </Text>
            </View>
          </View>
          <View
            bordered={true}
            footer={true}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderBottomWidth": 0.5,
                "borderColor": "#ccc",
                "borderRadius": 2,
                "borderTopWidth": 0.5,
                "flexDirection": "row",
                "paddingBottom": 10,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 10,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#007aff",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "600",
                }
              }
            >
              Version: 1.0
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </RCTSafeAreaView>
</View>
`;

exports[`PrivacyPolicy component renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      automaticallyAdjustContentInsets={false}
      contentContainerStyle={
        Array [
          Object {
            "padding": 10,
          },
          undefined,
        ]
      }
      contentInset={
        Object {
          "bottom": 0,
        }
      }
      enableAutomaticScroll={true}
      enableOnAndroid={false}
      enableResetScrollToCoords={true}
      extraHeight={75}
      extraScrollHeight={0}
      getScrollResponder={[Function]}
      handleOnScroll={[Function]}
      keyboardDismissMode="interactive"
      keyboardOpeningTime={250}
      keyboardShouldPersistTaps="handled"
      keyboardSpace={0}
      onScroll={[Function]}
      padder={true}
      resetKeyboardSpace={[Function]}
      resetScrollToCoords={
        Object {
          "x": 0,
          "y": 0,
        }
      }
      scrollEventThrottle={1}
      scrollForExtraHeightOnAndroid={[Function]}
      scrollIntoView={[Function]}
      scrollToEnd={[Function]}
      scrollToFocusedInput={[Function]}
      scrollToPosition={[Function]}
      showsVerticalScrollIndicator={true}
      style={
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        }
      }
      update={[Function]}
      viewIsInsideTabBar={false}
    >
      <View>
        <View
          style={
            Object {
              "backgroundColor": "#fff",
              "borderBottomWidth": 0.5,
              "borderColor": "#ccc",
              "borderLeftWidth": 0.5,
              "borderRadius": 2,
              "borderRightWidth": 0.5,
              "borderTopWidth": 0.5,
              "elevation": 3,
              "flexWrap": "nowrap",
              "marginBottom": 5,
              "marginLeft": 2,
              "marginRight": 2,
              "marginTop": 5,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.1,
              "shadowRadius": 1.5,
            }
          }
        >
          <View
            bordered={true}
            header={true}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderBottomWidth": 0.5,
                "borderColor": "#ccc",
                "borderRadius": 2,
                "flexDirection": "row",
                "paddingBottom": 15,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 15,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#007aff",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "600",
                }
              }
            >
              Privacy and Cookies Policy
            </Text>
          </View>
          <View
            bordered={true}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderBottomWidth": 0.5,
                "borderColor": "#ccc",
                "borderRadius": 2,
                "flexDirection": "row",
                "paddingBottom": 10,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 10,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "flex-start",
                  "alignSelf": "stretch",
                  "flex": 1,
                }
              }
            >
              <RCTSafeAreaView
                emulateUnlessSupported={true}
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "flex": 1,
                  }
                }
              >
                <RCTScrollView
                  automaticallyAdjustContentInsets={false}
                  contentContainerStyle={
                    Array [
                      Object {
                        "padding": undefined,
                      },
                      undefined,
                    ]
                  }
                  contentInset={
                    Object {
                      "bottom": 0,
                    }
                  }
                  enableAutomaticScroll={true}
                  enableOnAndroid={false}
                  enableResetScrollToCoords={true}
                  extraHeight={75}
                  extraScrollHeight={0}
                  getScrollResponder={[Function]}
                  handleOnScroll={[Function]}
                  keyboardDismissMode="interactive"
                  keyboardOpeningTime={250}
                  keyboardShouldPersistTaps="handled"
                  keyboardSpace={0}
                  onScroll={[Function]}
                  resetKeyboardSpace={[Function]}
                  resetScrollToCoords={
                    Object {
                      "x": 0,
                      "y": 0,
                    }
                  }
                  scrollEventThrottle={1}
                  scrollForExtraHeightOnAndroid={[Function]}
                  scrollIntoView={[Function]}
                  scrollToEnd={[Function]}
                  scrollToFocusedInput={[Function]}
                  scrollToPosition={[Function]}
                  showsVerticalScrollIndicator={true}
                  style={
                    Object {
                      "backgroundColor": "transparent",
                      "flex": 1,
                    }
                  }
                  update={[Function]}
                  viewIsInsideTabBar={false}
                >
                  <View>
                    <Text
                      style={
                        Object {
                          "color": "#000",
                          "fontFamily": "System",
                          "fontSize": 16,
                        }
                      }
                    >
                      1. This Privacy Policy informs about the methods of protection of personal data of the Website Users (" Users ") located at the electronic address www.naszdom.azurewebsites.net , hereinafter referred to as the " Website " pursuant to Regulation (EU) 2016/679 of the European Parliament and of the Council of 27 April 2016 on the protection of individuals with regard to the processing of personal data and on the free movement of such data, and repealing Directive 95/46 / EC (General Data Protection Regulation); (hereinafter " GDPR ") and on the general methods of using the Website, including the use of cookies.
                    </Text>
                    <Text
                      style={
                        Object {
                          "color": "#000",
                          "fontFamily": "System",
                          "fontSize": 16,
                        }
                      }
                    >
                      2. The administrator of the personal data of the Website Users ( hereinafter the "Administrator"; "ADO" ) is Palladia Limited Liability Company with its registered office in Warsaw (01-494) at ul. Obrońców Tobruku 21 B lok. 16, entered into the register of entrepreneurs of the National Court Register kept by the District Court for the Capital City of Warsaw in Warsaw, 12th Commercial Division of the National Court Register under the KRS number: 0000381211, with the NIP number: 527-265-15-01, REGON: 142869908, with the share capital of PLN 6,000.
                    </Text>
                    <Text
                      style={
                        Object {
                          "color": "#000",
                          "fontFamily": "System",
                          "fontSize": 16,
                        }
                      }
                    >
                      3. The content of the Website pages is the property of the Data Administrator and is legally protected.
                    </Text>
                    <Text
                      style={
                        Object {
                          "color": "#000",
                          "fontFamily": "System",
                          "fontSize": 16,
                        }
                      }
                    >
                      4. The website is informative. The administrator makes every effort to ensure that the presented data is as complete and up-to-date as possible.
                    </Text>
                    <Text
                      style={
                        Object {
                          "color": "#000",
                          "fontFamily": "System",
                          "fontSize": 16,
                        }
                      }
                    >
                      5. In order to use the Website, the User should have a computer or device with software installed that allows browsing websites and access to the Internet. Access to the Website may take place using the most popular web browsers.
                    </Text>
                    <Text
                      style={
                        Object {
                          "color": "#000",
                          "fontFamily": "System",
                          "fontSize": 16,
                        }
                      }
                    >
                      6. The Administrator declares that the Website pages are free from content that violates the rights of third parties or applicable law, in particular from content containing information that causes or threatens the privacy or security of any person, contains information promoting illegal activities or behavior that is offensive, threatening, indecent, defamatory or slanderous, inciting racism, persecution for ethnic, cultural or religious reasons, promoting or favoring criminal activities, violating the rights of third parties, including intellectual property rights or constituting another form of infringement of legally protected goods.
                    </Text>
                    <Text
                      style={
                        Object {
                          "color": "#000",
                          "fontFamily": "System",
                          "fontSize": 16,
                        }
                      }
                    >
                      7. Providing data is voluntary, however, their absence may prevent the use of the Website.
                    </Text>
                  </View>
                </RCTScrollView>
              </RCTSafeAreaView>
            </View>
          </View>
          <View
            bordered={true}
            footer={true}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderBottomWidth": 0.5,
                "borderColor": "#ccc",
                "borderRadius": 2,
                "borderTopWidth": 0.5,
                "flexDirection": "row",
                "paddingBottom": 10,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 10,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#007aff",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "600",
                }
              }
            />
          </View>
        </View>
      </View>
    </RCTScrollView>
  </RCTSafeAreaView>
</View>
`;

exports[`TermsUse component renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      automaticallyAdjustContentInsets={false}
      contentContainerStyle={
        Array [
          Object {
            "padding": 10,
          },
          undefined,
        ]
      }
      contentInset={
        Object {
          "bottom": 0,
        }
      }
      enableAutomaticScroll={true}
      enableOnAndroid={false}
      enableResetScrollToCoords={true}
      extraHeight={75}
      extraScrollHeight={0}
      getScrollResponder={[Function]}
      handleOnScroll={[Function]}
      keyboardDismissMode="interactive"
      keyboardOpeningTime={250}
      keyboardShouldPersistTaps="handled"
      keyboardSpace={0}
      onScroll={[Function]}
      padder={true}
      resetKeyboardSpace={[Function]}
      resetScrollToCoords={
        Object {
          "x": 0,
          "y": 0,
        }
      }
      scrollEventThrottle={1}
      scrollForExtraHeightOnAndroid={[Function]}
      scrollIntoView={[Function]}
      scrollToEnd={[Function]}
      scrollToFocusedInput={[Function]}
      scrollToPosition={[Function]}
      showsVerticalScrollIndicator={true}
      style={
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        }
      }
      update={[Function]}
      viewIsInsideTabBar={false}
    >
      <View>
        <View
          style={
            Object {
              "backgroundColor": "#fff",
              "borderBottomWidth": 0.5,
              "borderColor": "#ccc",
              "borderLeftWidth": 0.5,
              "borderRadius": 2,
              "borderRightWidth": 0.5,
              "borderTopWidth": 0.5,
              "elevation": 3,
              "flexWrap": "nowrap",
              "marginBottom": 5,
              "marginLeft": 2,
              "marginRight": 2,
              "marginTop": 5,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.1,
              "shadowRadius": 1.5,
            }
          }
        >
          <View
            bordered={true}
            header={true}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderBottomWidth": 0.5,
                "borderColor": "#ccc",
                "borderRadius": 2,
                "flexDirection": "row",
                "paddingBottom": 15,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 15,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#007aff",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "600",
                }
              }
            >
              Terms of Use
            </Text>
          </View>
          <View
            bordered={true}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderBottomWidth": 0.5,
                "borderColor": "#ccc",
                "borderRadius": 2,
                "flexDirection": "row",
                "paddingBottom": 10,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 10,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "flex-start",
                  "alignSelf": "stretch",
                  "flex": 1,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                  }
                }
              >
                1. The use of the app takes place using the ICT system. To use the Website, it is necessary to have a device with access to the Internet and a web browser.
              </Text>
              <Text
                style={
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                  }
                }
              >
                2. Before registering, the User or the Partner is required to read the Regulations, Privacy Policy and Cookies and accept them.
              </Text>
              <Text
                style={
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                  }
                }
              >
                3. The content of the app is the property of the Service Provider and is legally protected.
              </Text>
              <Text
                style={
                  Object {
                    "color": "#000",
                    "fontFamily": "System",
                    "fontSize": 16,
                  }
                }
              >
                4. The Service Provider provides the Service by enabling the publication of advertisements on the app's website.
              </Text>
            </View>
          </View>
          <View
            bordered={true}
            footer={true}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fff",
                "borderBottomWidth": 0.5,
                "borderColor": "#ccc",
                "borderRadius": 2,
                "borderTopWidth": 0.5,
                "flexDirection": "row",
                "paddingBottom": 10,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 10,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#007aff",
                  "fontFamily": "System",
                  "fontSize": 16,
                  "fontWeight": "600",
                }
              }
            />
          </View>
        </View>
      </View>
    </RCTScrollView>
  </RCTSafeAreaView>
</View>
`;
